src/h2load.cc

```diff
@@ -1199,6 +1319,12 @@ Worker::~Worker() {
   ev_loop_destroy(loop);
 }
 
+void Worker::stop_all_clients() {
+  for(auto client: clients) {
+    client->~Client();
```

Comment:
How does this work ownership-wise? It looks like the clients member is only keeping pointers, so someone must own the clients as well. Surely they will later also want to destruct the client?